docs/reference/ingest/processors/pipeline.asciidoc

```diff
@@ -7,7 +7,7 @@ Executes another pipeline.
 [options="header"]
 |======
 | Name             | Required  | Default  | Description
-| `name`           | yes       | -        | The name of the pipeline to execute
+| `name`           | yes       | -        | The name of the pipeline to execute. Supports <<accessing-template-fields,template snippets>>.
```

Comment:
have these links changed? `accessing-template-fields` points to [`common-options`](https://github.com/elastic/elasticsearch/blob/91d8ada4695f8395250eaae625f8ce7dfe57492d/docs/reference/ingest/processors/common-options.asciidoc) and `template snippets` to the `accessing template fields` paragraph on the `pipeline.asciidoc` page but there isn't such a paragraph

Reply:
I did a direct copy from the [set processor docs page ](https://www.elastic.co/guide/en/elasticsearch/reference/current/set-processor.html)and that links correctly. So I think it is good?

Reply:
Ah right, might be the github navigation that's not able to follow the links correctly.